packages/labs/analyzer/src/test/server/lit-html/template_test.ts

```diff
@@ -4,53 +4,48 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {suite} from 'uvu';
+import {test, describe as suite} from 'node:test';
 // eslint-disable-next-line import/extensions
 import * as assert from 'uvu/assert';
```

Comment:
do we also want to replace this with `chai@5` which supports esm?

Reply:
oh, node has an assert library too, I just forgot to use it!